### DISCUSSION TITLE
How about being able to reach each other on irc?

### DISCUSSION THREAD

**Discussion by ssbarnea** [Textualize/rich#357] [Ideas] [answered]
I spend a huge amount of time on freenode irc as most of my work is on various upstream projects.

Sometime we may find easier to chat about different subjects instead of spamming the issue tracker or discussions.

To avoid creating a new channel we could reuse `#pylib`, which is already multipurpose, covering pytest, tox and is not really busy.

I can be found on irc as `zbr` but if you mention my github username on that channel I will get a notification anyway.

**Comment by willmcgugan** [comment, marked as answer]
I would have no time for IRC. My attention is spread thin enough as it is.
